packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -110,6 +110,8 @@ Fetches a specific group by `id` from your organization
 
 Fetched [Group](#group-model)
 
+> This request is not supported from a browser and should be called server side.
```

Comment:
I'm not clear on what this means.

Reply:
I think better wording would be:
`This request is not supported from a browser and should not be called from there.`

If we're willing to share the technical details, which I suspect devs would find out on their own anyway, then we should say something along the lines of:
`The API returns a JSON response and if a browser doesn't mention that in the 'Accept Headers' then the API shouldn't be called from there`

Reply:
Hi @jakubvul @AshishVaidya-okta 

Modified the note to `This request is not supported from a browser and should not be called from there.`.

Reply:
Hi @jakubvul @AshishVaidya-okta 

Any updates for this PR?

Reply:
So is the problem essentially that you must include the `Accept: application/json` header in your request?

Reply:
Yes, that's my understanding following the comment from Brian Z. in the ticket. And that it happens for all our APIs. Are there reports of devs running into same problem for other APIs as the one here? Ideally, they should if browsers don't include `application/json` in the accept header. My understanding is that browsers don't request json by default -- https://developer.mozilla.org/en-US/docs/Web/HTTP/Content_negotiation/List_of_default_Accept_values.

So if devs can somehow add that to the accept header while calling from browsers then things will work, just as they do when hitting our API thru Postman. We still need to fix the error response though. It should be 406, and not 5xx.

Reply:
We tell people that they must include that header in our API overview page: https://developer.okta.com/docs/reference/api-overview/#media-types

It is also included in every sample request, including the one for this call.

Reply:
@jakubvul Removed the initial note and left only the grammar update.